offline.appcache

```diff
@@ -0,0 +1,44 @@
+CACHE MANIFEST
+# 2016-01-14 v1.1.3
+
+# Main HTML file
+index.html
+
+# Translations
+cz/index.html
+da/index.html
+de/index.html
+es/index.html
+fr/index.html
+ko/index.html
+pl/index.html
+pt/index.html
+ru/index.html
+zh/index.html
+
+# CSS Files
+css/critical.css
+css/full.css
+css/fonts/data-ttf.css
+css/fonts/data-woff.css
+css/fonts/data-woff2.css
+
+# JS Files
+assets/js/main.min.js
+assets/js/picturefill.min.js
+
+# Images
+assets/images/css-order-chart.png
```

Comment:
Well, since you're using picture, the browser will try to request the one fit for itself, but we can't asses that from the manifest, so we have to cache all of them so the one the browser one is there, right?